components/ui/ImageContainer.tsx

```diff
@@ -7,7 +7,7 @@ import Colors from "@/constants/color";
 export type ImageContainerProps = {
   images: string[];
   editable?: boolean;
-  onDelete?: (imageUrl: string) => void;
+  onDelete?: (image: string) => void;
```

Comment:
why do you need to change the wording here?

Reply:
Because, it not only imageUrl but also base64 to delete by this function so i decide to change wording kub

Reply:
Ok got it. You mean that the value passing here might be either url or base64